### DISCUSSION TITLE
Custom resource event type for Typescript

### DISCUSSION THREAD

**Discussion by paul-uz** [aws/aws-sdk-js-v3#3693] [Q&A]
I'm writing a small Lambda function in Typescript to run via a Custom Resource.

Is there a Type available for this event in Typescript? 

e.g for my API Gateway functions, I use the event type of `APIGatewayProxyEvent`

**Comment by aBurmeseDev** [comment]
Hi @paul-uz - sorry for not getting here sooner.

I'd need more context on what you're trying to accomplish with AWS Javascript SDK but it sounds like you're writing Lamba function with Typescript to run via Custom Resource. In that case, here's [related resource](https://docs.aws.amazon.com/lambda/latest/dg/typescript-handler.html#event-types) to check out. I'd be happy to assist further when you can elaborate more on the use case.

Best,
John
